charts/kubernikus-system/templates/prometheus-alerts.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: common-prometheus-rules
+  labels:
+    app: fusion
+    type: rule
+  annotations:
+    prometheus.io/rule: "true"
```

Comment:
What is the annotation good for?

Reply:
The operator picks up configmaps with this annotation